src/components/LoadingButton.js

```diff
@@ -25,5 +25,12 @@ const LoadingButton = ({ onClick, loading, ...props }) => {
     </div>
   );
 };
+LoadingButton.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  loading: PropTypes.bool,
+};
+LoadingButton.defaultProps = {
+  loading: undefined,
```

Comment:
```suggestion
  loading: null,
```

Reply:
I think this is a better pattern, requires a change above

Reply:
I agree, was just trying to avoid changing component code.